include/transformations/io-transformations/IOUtils.h

```diff
@@ -0,0 +1,14 @@
+#ifndef INCLUDE_TRANSFORMATIONS_IO_TRANSFORMATIONS_IOUTILS_H_
+#define INCLUDE_TRANSFORMATIONS_IO_TRANSFORMATIONS_IOUTILS_H_
+
+#include <string>
+#include <vector>
+#include <regex> // NOLINT
```

Comment:
Is everything so bad there?

Reply:
If I remove `NOLINT` I get the following warning generated by `cpplint`:
```<regex> is an unapproved C++11 header.  [build/c++11] [5]```